bin/import_fisher.py

```diff
@@ -171,9 +179,9 @@ def _split_sets(filelist):
     test_beg = dev_end
     test_end = len(filelist)
 
-    return filelist[train_beg:train_end],
-           filelist[dev_beg:dev_end],
-           filelist[test_beg:test_end]
+    return (pandas.DataFrame(data=filelist[train_beg:train_end], columns=columns),
+            pandas.DataFrame(data=filelist[dev_beg:dev_end],     columns=columns),
+            pandas.DataFrame(data=filelist[test_beg:test_end],   columns=columns))
```

Comment:
The result of slicing a DataFrame is itself a DataFrame, you don't need to reconstruct them here. This also eliminates the need for factoring out `columns`.